specs.js

```diff
@@ -112,75 +135,571 @@ describe('when using trimperson api', () => {
         done()
       })
     })
-    Then('it should be able to create a container', () => {
+    Then('it should be able to successfully use the function', (done) => {
       expect(errReturn).not.to.exist
       expect(dataReturn).to.exist
       createContainerMock.done()
+      done()
     })
   })
 
-  describe('when using the createRecord function', () => {
-    var title, containerId, extensionType, fileData, alternativeContainers, createRecordMock
+})
 
-    Given(() => title = 'inside-out')
-    Given(() => containerId = 'asdfjk')
-    Given(() => extensionType = '.mp4')
-    Given(() => alternativeContainers = ['himark'])
-    Given(() => fileData = 'data:video/mp4;base64,R0lGOD lhCwAOAMQfAP////7+/vj4+Hh4eHd3d/v7+/Dw8HV1dfLy8ubm5vX19e3t7fr')
+describe('When using trimperson api', () => {
+  var trim
+  var mockURL
+  var mockToken
 
-    describe('when the backend accepts the record', () => {
-      var errReturn
-      var dataReturn
-      Given(() => {
-        createRecordMock = nock(mockURL)
-          .filteringRequestBody(function (bodyt) {
-            return '*'
+  Given(() => mockURL = 'http://localhost')
+  Given(() => mockToken = 'DEADBEEF')
+
+  Given(() => {
+    trim = require('./index')({
+      url: mockURL,
+      token: mockToken,
+      mock: false
+    })
+  })
+
+  Then('it should have created a trimperson object', (done) => {
+    expect(trim).to.be.a('object')
+    done()
```

Comment:
Theres quite a bit of this. Not a biggie at all but you could remove quite a bit of 'done' boiler plate throughout the tests whenever async is not required